.env

```diff
@@ -13,3 +13,4 @@ PASSWORD=change_password
 DB=resources-psql
 FLASK_SKIP_DOTENV=1
 FLASK_ENV=development
+FLASK_DEBUG=1
```

Comment:
This, in conjunction with the change from `app` to `src` in docker-compose.yml allows live-reloading while developing